web/client/localConfig.json

```diff
@@ -733,8 +733,25 @@
           "nav": false,
           "menuProps": {
               "noCaret": true
-          }
-      }
+					},
+					"toolsCfg": [{
+								"buttonSize": "small",
+								"includeCloseButton": false,
+								"useModal": false,
+								"closeGlyph": "1-close"
+							}, {
+								"buttonSize": "small",
+								"includeCloseButton": false,
+								"useModal": false,
+								"closeGlyph": "1-close"
+							}, {
+								"buttonSize": "small",
+								"includeCloseButton": false,
+								"useModal": false,
+								"closeGlyph": "1-close"
+					 }]
```

Comment:
had to change the localConfig to get dialogs also in the home, (we can just return this renderDialog() in the components)